README.md

```diff
@@ -63,13 +63,14 @@ sudo pacman -U apparmor.d-*.pkg.tar.zst \
 
 > **Warning**: for a first install, it is recommanded to install all profiles in complain mode. See [Complain mode](#troubleshooting)
 
-**Debian**
+**Debian / Ubuntu**
 
 Build using standard Debian package build tools:
 ```sh
-sudo apt install apparmor-profiles build-essential config-package-dev debhelper golang-go rsync
+sudo apt install apparmor-profiles build-essential config-package-dev debhelper golang-go rsync git
+git clone https://github.com/roddhjav/apparmor.d.git && cd apparmor.d
 dpkg-buildpackage -b -d --no-sign
-sudo dpkg -i ../apparmor.d_*_all.deb
+sudo dpkg --force overwrite -i ../apparmor.d_*_all.deb
```

Comment:
```
dpkg: error processing archive ../apparmor.d_0.001-1_all.deb (--install):
 trying to overwrite '/etc/apparmor.d/abstractions/lightdm', which is also in package lightdm 1.26.0-7
dpkg-deb: error: paste subprocess was killed by signal (Broken pipe)
Errors were encountered while processing:
 ../apparmor.d_0.001-1_all.deb
```

Reply:
This is not the correct way of doing this. You should add `/etc/apparmor.d/abstractions/lightdm.apparmor.d` to `debian/apparmor.d.displace`

Reply:
Hmm, that would be the same error.

Reply:
It should work fine. This is how `/etc/apparmor.d/tunables/xdg-user-dirs` is displaced.

Now, in this context, the solution is to remove `abstractions/lightdm` and `apparmor.d/abstractions/lightdm_chromium-browser` from this project are we are not modifying them. These files were added by morfikov, but since my fork evolved, we don't overwrite other package profiles unless it is strictly required.